src/index.html

```diff
@@ -13,6 +13,38 @@
     />
   </head>
   <body>
-    <h1>Calendar</h1>
+    <div class="calendar calendar--start-day-sun calendar--month-length-31">
+      <div class="calendar__day"></div>
```

Comment:
In this case, you have a duplicate element inside ( div with calendar__day class). It's a better way to use a list element for this.